ui/src/components/main.tsx

```diff
@@ -102,6 +103,7 @@ export class Main extends Component<any, MainState> {
     dataType: getDataTypeFromProps(this.props),
     sort: getSortTypeFromProps(this.props),
     page: getPageFromProps(this.props),
+    hasUnsavedData: false,
```

Comment:
Its better if you clena this up yourself. Just delete the lines, make a new commit and push it to the same branch.

Reply:
Fixed